src/mapd-con-es6.js

```diff
@@ -285,6 +278,18 @@ class MapdCon {
     return this
   }
 
+  removeConnection(conId) {
+    if (conId < 0 || conId >= this.numConnections) {
```

Comment:
How sure are we that `numConnections` is accurate? My instinct here is to directly check the length of `_client` and `_sessionId`, but if we're absolutely sure `numConnections` is accurate, then that may not be necessary.

Reply:
This function I didn't change at all, I merely moved it to be closer to where it makes sense (next to the other connection methods). And I don't see any place we're even using this in Immerse today.

Reply:
That said, you're right, and I'd probably do it that way if we were to redo this or add to it.